### PR TITLE
fix: remove clang-format dependency

### LINKED ISSUE #97: npm install fails on travis testing with node 6
`npm install` is failling with the following error during travis testing when run with node 6:

```
35.69s$ npm install 
npm ERR! Linux 4.9.6-040906-generic
npm ERR! argv "/home/travis/.nvm/versions/node/v6.12.2/bin/node" "/home/travis/.nvm/versions/node/v6.12.2/bin/npm" "install"
npm ERR! node v6.12.2
npm ERR! npm  v3.10.10
npm ERR! code EREADFILE
npm ERR! Error extracting /home/travis/.npm/clang-format/1.0.40/package.tgz archive: ENOENT: no such file or directory, open '/home/travis/.npm/clang-format/1.0.40/package.tgz'
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>
npm ERR! Please include the following file with any support request:
npm ERR!     /home/travis/build/GoogleCloudPlatform/cloud-profiler-nodejs/npm-debug.log
```

### PR DESCRIPTION
**Pull request by nolanmar511**
fixes #97

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
     "@types/pretty-ms": "^3.0.0",
     "@types/request": "^2.0.7",
     "@types/sinon": "^4.0.0",
-    "clang-format": "^1.0.33",
+    "clang-format": "^1.0.41",
```
Comment (ofrobots) [inline package.json]: Why does this module depend on `clang-format` in the first place?

Comment (nolanmar511) [reply package.json]: Not sure when dependency was added -- but I was able to remove this without problem.

**Comment by ofrobots** [comment]
I think the root cause of #97 is related to this fix in `gts`: https://github.com/google/ts-style/pull/100. That fix is contained in the just published `gts@0.5.3`. It locks down the `clang-format` dependency to avoid breakages like #97.

Perhaps you can upgrade the `gts` version here.

**Comment by nolanmar511** [comment]
I've made a separate PR to update the gts dependency (#99)

**Comment by nolanmar511** [comment]
PTAL

**Pull request merged**